doc/developer-guides/hld/hld-devicemodel.rst

```diff
@@ -818,8 +822,8 @@ Functions implemented by ACPI include:
 -  Battery management
 -  Thermal management
 
-All critical functions depends on ACPI tables.
-On an APL platform with Linux installed we can see these tables using:
+All critical functions depend on ACPI tables.
+On an Apollo Lake platform with Linux installed, we can see these tables using:
```

Comment:
Appolo Lake is no longer supported. Can we update this to show the output on Tiger Lake
/cc @NanlinXie

Reply:
> Appolo Lake is no longer supported. Can we update this to show the output on Tiger Lake /cc @NanlinXie

Suggest we create an issue ticket for future.

Reply:
OK, that works for me! (Tracked on #6861 )